cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -461,15 +456,15 @@ func (sub EthereumSub) logToEvent(clientChainID *big.Int, contractAddress common
 
 // handleEthereumEvent unpacks an Ethereum event, converts it to a ProphecyClaim, and relays a tx to Cosmos
 func (sub EthereumSub) handleEthereumEvent(events []types.EthereumEvent) error {
-	var prophecyClaims []ethbridge.EthBridgeClaim
+	var prophecyClaims []*ethbridge.EthBridgeClaim
```

Comment:
Why does this array of EthBridgeClaims have to be an array of pointers? cc @iverc @timlind

Reply:
Why can't it just be an array of EthBridgeClaims?